platforms/chibios/drivers/i2c_master.c

```diff
@@ -129,7 +149,11 @@ __attribute__((weak)) void i2c_init(void) {
         palSetLineMode(I2C1_SDA_PIN, PAL_MODE_INPUT);
 
         chThdSleepMilliseconds(10);
-#if defined(USE_GPIOV1)
```

Comment:
Similar concerns here to other MCU support PRs, where the differences are not being abstracted and just adding to the walls of PP checks.

Reply:
Done! All i2c driver adjustments are now handled via `chibios_config.h`